### PR TITLE
Upgrade rails to 6.1.7.4

### PR DESCRIPTION
**Pull request by eduardoborba**
Auto-created Jira Issue: https://doximity.atlassian.net/browse/APPSEC-1556

---

Upgrade all apps on rails 6 to 6.1.7.4 to fix yet another security vulnerability. [reference](https://rubyonrails.org/2023/6/26/Rails-Versions-7-0-5-1-6-1-7-4-have-been-released)

---

QA: Smoke test the app

[_Created by Sourcegraph batch change `eduardoborba/rails-6-upgrade`._](https://sourcegraph.build.dox.pub/users/eduardoborba/batch-changes/rails-6-upgrade)

### CODE REVIEW

**Comment by eduardoborba** [comment]
doxbot batch APPSEC-1430

**Comment by eduardoborba** [comment]
doxbot cr

File: Gemfile [hunk 1]
```diff
@@ -9,4 +9,4 @@ group :development do
 end
 
 gem "pry", group: [:development, :test], require: false
-gem "rails", "~> 6.1.7.3", group: [:development, :test], require: false
+gem "rails", "~> 6.1.7.4"
```
Comment (keatly91) [inline Gemfile:12]: Lets keep the group here.

```suggestion
gem "rails", "~> 6.1.7.4", group: [:development, :test], require: false
```

Comment (eduardoborba) [reply Gemfile:12]: Good catch

**Review by keatly91 — APPROVED** [review]

**Pull request merged**